AdminUi/src/AdminUi/Properties/.gitignore

```diff
@@ -0,0 +1 @@
+launchSettings.json
```

Comment:
Why do you gitignore this file?

Reply:
The file is not used when launching the AdminUi from Docker. Keeping it out makes this clearer.